### PR TITLE
C cpp

### LINKED ISSUE #1947: [C/CPP]: Input Output operators in C++using cin and cout
### Description

This will focus on the operators used to take input and provide output in C++ programs and the header files used in the process.

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Sunainacode**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #1947 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: C_CPP/Input & Output Operations/cin_cout-in-cpp.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Output: This is a code snippet indicating&emsp; tab space
+
+Note: The cin and cout can contain multiple pairs of angular brackets for taking multiple input and printing multiple output using a single line.
+
+```
+cin>>a>>b>>c;
+cout<<a<<"\n"<<b+c;
+```
+Input:
+3 4 5
```
Comment (deeqakkk) [inline C_CPP/Input & Output Operations/cin_cout-in-cpp.md:63]: add a <br> tag here

**Review by deeqakkk — CHANGES_REQUESTED** [review]
make that one change, and you are good to go.

File: C_CPP/Input & Output Operations/cin_cout-in-cpp.md (hunk 1 above)
Comment (Sunainacode) [reply C_CPP/Input & Output Operations/cin_cout-in-cpp.md:63]: sir i am not able to understand. can you please explain what i have to add as a?

Comment (apeksha235) [reply C_CPP/Input & Output Operations/cin_cout-in-cpp.md:63]: @Sunainacode he means to say add a break tag

Comment (Sunainacode) [reply C_CPP/Input & Output Operations/cin_cout-in-cpp.md]: Okay Done

**Pull request merged**